build.gradle

```diff
@@ -1,30 +1,44 @@
+plugins {
+    id "com.jfrog.artifactory" version "4.20.0"
+}
+
 apply plugin: 'groovy'
 apply plugin: 'maven-publish'
 apply plugin: 'maven'
 apply plugin: 'java-gradle-plugin'
 
 group = 'net.uberfoo.gradle'
-version = '2.6-SNAPSHOT'
+version = '2.6-sig1-SNAPSHOT'
```

Comment:
This is still a Snapshot, I fixed everything so we can release from a local machine. 

` @Reviewer`:
**TO BE DISCUSSED: How to wire up publishing with CI.**

Reply:
This bit could be a separate PR / Jira issue, though.

Reply:
Discussed with Tommy: separate PR